bdd/features/e2e/E2E-010-cli.feature

```diff
@@ -63,19 +63,6 @@ This feature checks CLI functionalities
         Then I wait for Instance health status to change from 200 to 404
         And host is still running
 
-    # Test E2E-010 TC-010 works but it is ignored, because changes need to be made in CLI to end the displayed stream.
-    @ignore
-    Scenario: E2E-010 TC-006 Get log from Instance
-        Given host is running
-        Then I set json format
-        Then I use apiUrl in config
-        When I execute CLI with "seq send ../packages/reference-apps/hello-alice-out.tar.gz" arguments
-        Then I get Sequence id
-        Then I start Sequence
-        And I get Instance id
-        Then I get Instance log
-        And host is still running
-
```

Comment:
This test refers to CLI command `si inst log <id>`. I think it is nice to have. It works after small refactor:
```
    @ci @cli
    Scenario: E2E-010 TC-006 Get log from Instance
        Given host is running
        Then I set json format
        Then I use apiUrl in config
        When I execute CLI with "seq send ../packages/reference-apps/inert-function.tar.gz" arguments
        Then I get Sequence id
        Then I start Sequence
        And I get Instance id
        When I execute CLI with "inst log -" arguments
        And host is still running
```
Executing the log test locally:

![image](https://user-images.githubusercontent.com/61054769/168549005-9b83fa58-7c8f-4877-bd79-abdaaff46122.png)

Reply:
I will try to revive it in a separate PR, to make sure it works on CI.

Reply:
Good idea!